hw02/src/cs3500/hw02/LaxCoinGameModel.java

```
@@ -0,0 +1,20 @@
+package cs3500.hw02;
+
+public final class LaxCoinGameModel extends AbstractCoinGameModel {
+
+  public LaxCoinGameModel(String board) {
+    super(board);
+  }
+
+  @Override
+  protected final void checkMoveLegality(int coinPosition, int newPosition) {
```

Why are you throwing an exception via that method instead of returning a bool?